python/tester/upload_from_repo.py

```diff
@@ -360,7 +360,8 @@ def _upload_one_image(temp_dir, group_id, butler, ref):
                             Body=json.dumps(dict(hdul[0].header)),
                             Key=dest_key.removesuffix("fits") + "json",
                         )
-            dest_bucket.upload_file(path, dest_key)
+            with open(path, "r") as temp_file:
```

Comment:
This looks like the file is in read mode, not write.

Reply:
Yes, the temporary file is in read mode here as it's being uploaded to the object store.